src/components/LuxAutocompleteInput.vue

```diff
@@ -218,6 +219,9 @@ export default {
         this.arrowCounter = -1
       }
     },
+    onClickResult(result) {
+      this.setResult(result, this.emitSelectedId)
```

Comment:
We should just call emit selected here after calling set result.  Would simplify the code a bit

Reply:
I see we do not have the result with the id, but the label here.